js_modules/dagster-ui/packages/ui-core/src/app/AppProvider.tsx

```diff
@@ -102,6 +103,19 @@ const GlobalStyle = createGlobalStyle`
     font-family: ${FontFamily.monospace};
     font-size: 16px;
   }
+
+  :focus-visible {
+    outline: ${colorFocusRing()} auto 1px;
+  }
+
+  :focus:not(:focus-visible) {
+    outline: none;
+  }
+
+  :not(a):focus,
+  :not(a):focus-visible {
+    outline-offset: 1px;
+  }
```

Comment:
Apply our `colorFocusRing` throughout the app on all focusable items, overriding the light blue default provided by the browser.